components/BarcodeScanner.vue

```diff
@@ -305,7 +305,9 @@ export default {
               this.$emit('error', message);
               return;
             }
-            this.$emit('result', this.getDLInfo(results[0].barcodeText));
+            this.$emit('result', this.getDLInfo({
+              txt: results[0].barcodeText, bytes: results[0].barcodeText
```

Comment:
This looks like we're passing the same thing (`results[0].barcodeText`) as both `txt` and `bytes`. A typo maybe?

Reply:
Confusingly it was both a typo and the right thing to do. `results[0].barcodeTxt` contains the information we need.